macros/multiple_databases/string_agg.sql

```diff
@@ -14,6 +14,12 @@
     {%- else -%}
         nullif(string_agg(convert(nvarchar(2000), {{ string_field}}), ', '), '')
     {%- endif -%}
+{%- elif target.type == 'databricks' -%}
+    {%- if delimiter is defined -%}
+        concat_ws('{{ delimiter }}', collect_list(cast({{ string_field }} as string)))
+    {%- else -%}
+        concat_ws(', ', collect_list(cast({{ string_field }} as string)))
```

Comment:
- The cast to string should not be necessary, as the input should already be a string
- There should be a nullif() to catch the situation where have all `null` values